wasm/src/lsp/parse.rs

```diff
@@ -1,10 +1,19 @@
-use std::collections::HashMap;
+use std::borrow::Borrow;
+use std::collections::{HashMap, HashSet};
 use std::rc::Rc;
-use clvm_tools_rs::compiler::comptypes::{BodyForm, CompileErr, CompileForm};
-use clvm_tools_rs::compiler::sexp::SExp;
+
+use lsp_types::Position;
+
+use clvm_tools_rs::compiler::compiler::DefaultCompilerOpts;
+use clvm_tools_rs::compiler::comptypes::{BodyForm, CompileErr, CompileForm, HelperForm, LetData, LetFormKind};
+use clvm_tools_rs::compiler::frontend::frontend;
```

Comment:
This also should be annotated with `#[cfg(test)]`

Reply:
Thanks.